client/src/components/EstatePage.css

```diff
@@ -143,3 +143,34 @@
   margin-top: 12px;
   font-size: smaller;
 }
+
+.propertyCard {
+  width: 345px;
+  margin: 0 auto;
+  margin-top: 25px;
+  box-shadow: 0px 0px 5px 0px rgb(85, 85, 85);
+  color: #707070;
+  font-weight: 100;
+  background-color: #dceff8;
+  padding-bottom: 1px;
+}
+
+.p {
+  margin: 5px;
+}
+
+.linkBox {
+  text-decoration: none !important;
```

Comment:
Why is `!important` so important here? Maybe you can find another way of increasing the specificity of this ruleset
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity